pact/policy-manager/policy-manager.repl

```diff
@@ -157,8 +163,7 @@
 
     @doc "Concrete policy for issuing an nft with a fixed supply of 1"
 
-    (defcap GOVERNANCE ()
-      (enforce-guard (keyset-ref-guard 'marmalade-admin )))
+    (defcap GOVERNANCE () true)
```

Comment:
That's only a test module in the REPL script. Not a "real" module
It's only purpose is to test the functions `(write-concrete-policy)` and  `(get-concrete-policy)` of the policiy manager.

My idea is make as simple as possible , and to avoid any possible pain.